tox.ini

```diff
@@ -1,23 +1,24 @@
 [tox]
-envlist = py27,py27-simplejson,py34
+envlist = {py27,py35,py36}-{default,simplejson},pre-commit,cover,docs
```

Comment:
No it is not the list of available environments, it's the list of environments that are executed by default, i.e. when we do `make test`. Unless there is a reason to build docs by default we should remove it from here. `tox -e docs` will still continue to work.

Also note that you're running tests once more just for coverage.